examples/rdkafka_example.c

```diff
@@ -187,6 +189,27 @@ static void msg_consume (rd_kafka_message_t *rkmessage,
 				!timestamp ? 0 :
 				(int)time(NULL) - (int)(timestamp/1000));
 		}
+
+                if (!rd_kafka_message_headers(rkmessage, &hdrs)) {
+                        size_t idx = 0;
+                        const char *name;
+                        const void *val;
+                        size_t size;
+
+                        fprintf(stdout, "%% Headers:");
+
+                        while (!rd_kafka_header_iter_all(hdrs, idx++,
```

Comment:
I'd remove the word `iter` from this method name since you're iterating outside the method call (with the `idx++`).  also the word `all`. perhaps just `rd_kafka_header_get` or something?

Reply:
Okay, what about `rd_kafka_header_get()` for the key-specific one, and `rd_kafka_header_enum()` or `..list()` or `..get_all()` for the one that iterates all keys?